src/components/Workshop/getUpgradesAvailable.js

```diff
@@ -21,7 +20,9 @@ export function getUpgradesAvailable({ learnedForgeRecipes, toolLevels }) {
       for (let ingredient of Object.keys(nextLevelUpgrade.ingredients)) {
         allIngredientsUnlocked =
           allIngredientsUnlocked &&
-          !!(!recipesMap[ingredient] || learnedRecipeIds.includes(ingredient))
+          !!(
+            !recipesMap[ingredient] || learnedForgeRecipes.includes(ingredient)
+          )
```

Comment:
Looks like this is just a Prettier change, yeah?

Reply:
yep! though coming back to this line i found it confusing so it would be nice to refactor for readability in the future